packages/eslint-config-kibana/.eslintrc.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
       'error',
       {
         endOfLine: 'auto',
+        arrowParens: 'avoid',
```

Comment:
Should we keep the BWC with the current style?

Reply:
I think we should avoid adding prettier configuration at this time, lets pull the bandaid off and move forward with prettier.